.github/workflows/wheels-publish.yaml

```diff
@@ -41,9 +41,9 @@ jobs:
     name: wheels publish
     runs-on: linux-amd64-cpu4
     container:
-      # ctk version of the container is irrelevant in the publish step
-      # it's simply a launcher for twine
-      image: "rapidsai/ci-wheel:cuda12.0.1-centos7-py3.10"
+      # CUDA toolkit version of the container is irrelevant in the publish step.
+      # This just uploads already-built wheels to remote storage.
```

Comment:
This workflow no longer uses `twine`, as of #154.

Proposing changing this comment to something correct and more future-proof.